bin/check-mesos.rb

```diff
@@ -29,29 +29,26 @@
 require 'sensu-plugin/check/cli'
 require 'rest-client'
 
-# Mesos default ports are defined here: http://mesos.apache.org/documentation/latest/configuration
-MASTER_DEFAULT_PORT = '5050'.freeze
-SLAVE_DEFAULT_PORT = '5051'.freeze
-
 class MesosNodeStatus < Sensu::Plugin::Check::CLI
   option :server,
          description: 'Mesos servers, comma separated',
          short: '-s SERVER1,SERVER2,...',
          long: '--server SERVER1,SERVER2,...',
          default: 'localhost'
 
-  option :mode,
```

Comment:
not in `bin/metrics-mesos.rb`, in the other scripts I removed the `--mode` option because it's not used anymore, in `bin/metrics-mesos.rb` it's used to name the metric and it's mandatory, then i have:
```
    case config[:mode]
    when 'master'
      port = config[:port] || MASTER_DEFAULT_PORT
    when 'slave'
      port = config[:port] || SLAVE_DEFAULT_PORT
    end
```
if `config[:port]` is defined it will be used regardless of the `mode` if not, `mode` and the constants will be used to determine the default port insted of using a single default in the option definition.